pydax/_high_level.py

```diff
@@ -184,6 +212,32 @@ def get_dataset_metadata(name: str, *,
     :param human: Whether to return the metadata as a string in human-readable form or to return a copy of the
         dataset's schema. Defaults to True.
     :return: Return a dataset's metadata either as a string or as a schema dictionary.
+
+    Example get metadata as string:
+
+    >>> metadata = get_dataset_metadata('gmb')
+    >>> print(metadata)  # doctest:+ELLIPSIS
+    Dataset name: Groningen Meaning Bank Modified
+    Description: A dataset of multi-sentence texts, together with annotations for parts-of-speech...
+    Size: 10M
+    Published date: 2019-12-19
+    License: Community Data License Agreement – Sharing, Version 1.0 (CDLA-Sharing-1.0)
+    Available subdatasets: gmb_subset_full
+
+    Example get metadata as dict
```

Comment:
```suggestion
    Example get metadata as dict:
```

Is this ready for review? The PR looks good and this is the only comment I have.

Reply:
Yeah it's ready.